Packages/vcs/vcs/vcsvtk/isolinepipeline.py

```diff
@@ -47,11 +53,19 @@ def _plotInternal(self):
 
         if len(linewidth) < len(self._contourLevels):
             # fill up the line width values
-            linewidth += [1.0] * (len(self._contourLevels) - len(linewidth))
+            if (len(linewidth) == 0):
```

Comment:
@danlipsa looks to me that this code is repetitive. Would it be useful to have a method in the same class?